### ISSUE TITLE
react-native link react-native-oauth fails on Windows

### ISSUE THREAD

**Issue by Cleanshooter** [fullstackreact/react-native-oauth#50]
It seems the addition of the prelink and postlink rnpm commands to the json.package prevent Windows users from using this tool.  The shell scripts you have written are not windows env compatible and cause rnpm-install to crash. I'll see if I can write a fix for this.  

```
rnpm-install ERR! It seems something went wrong while linking. Error: spawn UNKNOWN
Please file an issue here: https://github.com/facebook/react-native/issues

spawn UNKNOWN
```

**Comment by Cleanshooter** [comment]
Removing the following from package.json allows linking to complete succesfuly.

```
  "rnpm": {
    "commands": {
      "prelink": "node_modules/react-native-oauth/bin/prepare.sh",
      "postlink": "node_modules/react-native-oauth/bin/cocoapods.sh"
    },
    "ios": {
      "project": "ios/OAuthManager.xcodeproj"
    },
    "android": {
      "packageInstance": "new OAuthManagerPackage()"
    }
  },
```
[https://github.com/Cleanshooter/react-native-oauth](https://github.com/Cleanshooter/react-native-oauth)

Not really a solution but a work around for now... I guess.  I'm not experienced enough with npm packages to figure out how to run shells from Windows CMD so this library would be more cross compatible.

**Comment by gbenga504** [comment]
Nice....  That worked for me also

**Comment by roshkins** [comment]
I had the same issue, and the workaround worked! Could executing .sh files under Bash on Ubuntu on Windows work?

**Comment by Cleanshooter** [comment]
Yeah your best best for this to work as is might be to run it in a docker container or something...

**Comment by KeanuTomatoMomotaro** [comment]
@Cleanshooter i tried the solution and it linking was successful, however i was unable to build and run the application as I got the following error:
```
Execution failed for task ':react-native-oauth:compileReleaseJavaWithJavac'.
```

**Comment by jinz11** [comment]
manually removed this section will cause app failed to generate apk due to verify release resource step failed. I think should update this changes on repo instead of remove it manually.
